apps/studio/src/lib/editor.ts

```diff
@@ -77,12 +89,10 @@ export function makeDBHint(
   };
 }
 
-const quot = "'\"`";
-
 const SCHEMA_TABLE = new RegExp(
-  "(?:(?<schema>[quot]?.+?[quot]?)\\.(?<table1>[quot]?.+[quot]?)|(?<table2>[quot]?.+[quot]?))".replaceAll(
-    "quot",
-    quot
+  "(?:(?<schema>[quote]?.+?[quote]?)\\.(?<table1>[quote]?.+[quote]?)|(?<table2>[quote]?.+[quote]?))".replaceAll(
+    "quote",
+    "'\"`"
   )
 );
```

Comment:
@azmy60 As I'm looking at this again, I'm wondering what the `table2` capture group is for here. Maybe I'm just bad at regex, but it seems to me like `table1` will always be the group that matches, yes?

Reply:
Nevermind I'm dumb lol